CMakeLists.txt

```diff
@@ -175,10 +175,14 @@ if (NANOGUI_BUILD_SHARED)
 endif()
 
 # Always use libc++ on Clang
-if (CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+if (CMAKE_CXX_COMPILER_ID MATCHES "Clang" AND NOT ${CMAKE_SYSTEM_NAME} MATCHES "BSD")
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -stdlib=libc++")
```

Comment:
On OpenBSD and FreeBSD, this will raise a warning ``clang-3.8: warning: argument unused during compilation: '-stdlib=libc++'``